documentation/modules/metrics/archive_metrics-dashboard-descriptions.adoc

```diff
@@ -0,0 +1,211 @@
+
+// This assembly is included in the following assemblies:
+//
+// metrics/assembly-metrics-config-files.adoc
+
+[id='ref-metrics-dashboards-{context}']
+= Example Grafana dashboards
+
+[role="_abstract"]
+If you deploy Prometheus to provide metrics,
+you can use Strimzi's example Grafana dashboards to monitor Strimzi components.
```

Comment:
Again, I wonder if we should be a bit more open to other tools. Our examples use Prometheus and Grafana, but other compatible tools can be used as well.

Reply:
I added a note, but this file is archived and not included in the doc. 
This files isn't used now.